right/src/keyboard_layout.c

```diff
@@ -82,6 +82,7 @@ bool handleKey(uhk_key_t key, int scancodeIdx, usb_keyboard_report_t *report, co
         if (key_toggled_off(prevKeyStates, currKeyStates, keyId)) {
             ActiveLayer = LAYER_ID_BASE;
         }
+        LedDisplay_SetLayerLed(ActiveLayer, 0xff);
```

Comment:
Let's have a global `LedDisplayBrightness` value and reference it here in place of 0xff. Later on we'll be able to reuse it elsewhere.